config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
-    get "/help", to: "static_pages#help"
-    get "/about", to: "static_pages#about"
-    get "/contact", to: "static_pages#contact"
-    get "/signup", to: "users#new"
+    get "/help",      to: "static_pages#help"
```

Comment:
ửa tương tự bên dưới
```suggestion
    get "/help", to: "static_pages#help"
```